src/java.base/share/classes/jdk/internal/foreign/abi/AbstractLinker.java

```diff
@@ -66,7 +74,7 @@ public abstract sealed class AbstractLinker implements Linker permits LinuxAArch
                                                                       Windowsx64Linker, AixPPC64Linker,
                                                                       LinuxPPC64Linker, LinuxPPC64leLinker,
                                                                       LinuxRISCV64Linker, LinuxS390Linker,
-                                                                      FallbackLinker {
+                                                                      ZosS390Linker, FallbackLinker {
```

Comment:
My suggestion was to put `ZosS390Linker` on a separate line (so we don't change the line for `FallbackLinker`).

Reply:
Fixed.